projects/plugins/jetpack/modules/carousel/jetpack-carousel.php

```diff
@@ -688,9 +689,11 @@ class_exists( 'Jetpack_AMP_Support' )
 		$large_file_info = wp_get_attachment_image_src( $attachment_id, 'large' );
 		$large_file      = isset( $large_file_info[0] ) ? $large_file_info[0] : '';
 
-		$attachment       = get_post( $attachment_id );
-		$attachment_title = wptexturize( $attachment->post_title );
-		$attachment_desc  = wpautop( wptexturize( $attachment->post_content ) );
+		$attachment			 = get_post( $attachment_id );
```

Comment:
Should we check that `get_post` isn't null before we start pulling data from `$attachment`?

You'll also need to address the indenting issues so we can merge this.

Reply:
This is old code, I'll add a check here though.